.github/workflows/test.yml

```diff
@@ -17,5 +17,14 @@ jobs:
         with:
           go-version: 1.19
           check-latest: true
+      - name: Setup node
+        uses: actions/setup-node@v3
```

Comment:
can you please use the git hash for the latest release? similar we have for the others, thanks!